### ISSUE TITLE
Add more related methods.

### ISSUE THREAD

**Issue by suryapratapsinghsuryavanshi** [suryapratapsinghsuryavanshi/solverjs#32]
If you are a beginner to open source this repository gives you a great learning experience. Please contribute more related methods. Please reach out if you find any inconvenience.

Email: _dev@suryapratap.ml_

**Comment by ayush3160** [comment]
Hi @suryapratapsinghsuryavanshi, should I go for a strong password generator. Which will take password length and will generate a random strong password.

**Comment by suryapratapsinghsuryavanshi** [comment]
> Hi @suryapratapsinghsuryavanshi, should I go for a strong password generator. Which will take password length and will generate a random strong password.

Yes, If your logic is different from the existing method or if generates a strong and unique string than the existing one.
Existing Method: https://github.com/suryapratapsinghsuryavanshi/solverjs#token
https://github.com/suryapratapsinghsuryavanshi/solverjs/blob/06edf43a8074a71d7b410035e097ed83dda3d36d/src/math/math.js#L457
